numba_cuda/numba/cuda/cudadrv/nvrtc.py

```diff
@@ -289,7 +289,7 @@ def get_lto(self, program):
         return lto
 
 
-def compile(src, name, cc, ltoir=False):
+def compile(src, name, cc, ltoir=False, lineinfo=False, debug=False):
```

Comment:
It's a bit of an oversight that we never passed these flags to NVRTC before - have you noticed if this now enables debugging / profiling of NVRTC-compiled code used in Numba kernels with these changes?

Reply:
There was a discussion offline on this. I think this will serve as part of the solution.